src/notification-banner/index.js

```diff
@@ -0,0 +1,3 @@
+import NotificationBanner from './NotificationBanner';
+
+export default NotificationBanner;
```

Comment:
This doesn't matter, but you could shorten this line
```suggestion
import { default as NotificationBanner } from './NotificationBanner';

```

Reply:
hold on i got this backwards

Reply:
there might be a way to one line this but i've already spent too much time on it 😁